### DISCUSSION TITLE
Anti-Flash Patch, Server Say Command, and Another Entrance/Exit Commands?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#339] [Ideas]
Basically disable the white screen when you got flashbanged.
Create a /say where the text is blue like the joined button to troll XD.
And is it possible to add another /exit or /enter command so we can go to fire exit doors :3

**Comment by joep26020** [comment]
> Basically disable the white screen when you got flashbanged. Create a /say where the text is blue like the joined button to troll XD. And is it possible to add another /exit or /enter command so we can go to fire exit doors :3

just do /announce for your second q.

**Reply by Dexteriority** [reply]
Got it

**Comment by winstxnhdw** [comment]
I am going to keep these as low-priority. We've been adding so many features aggressively for the past month and I think it's time we focus on some stability.

**Reply by Dexteriority** [reply]
Stability? may i know what that means? since i think the hax are pretty stable right now 🤔

**Reply by winstxnhdw** [reply]
In software, “just working” is really the bare minimum. It’s often as important or more important that the project is robust and maintainable. These are usually features that you as the user won’t notice in the short-term.

**Reply by Dexteriority** [reply]
ah ok
